composer.json

```diff
@@ -32,6 +32,7 @@
         "knplabs/gaufrette": "^0.1.6 || ^0.2 || ^0.3",
         "kriswallsmith/buzz": "^0.15",
         "psr/log": "^1.0",
+        "sonata-project/block-bundle": "^3.11",
```

Comment:
Why is this a required dep now?

Reply:
@core23 with `sonata-project/block-bundle < 3.11` service `sonata.templating` will not be available. There are some bundles, where i added this dependency. And @jordisala1991 wrote the same about PR for [NewsBundle](https://github.com/sonata-project/SonataNewsBundle/pull/428#pullrequestreview-91079410).

If adding new dependency is wrong, please tell me what do i have to do, see https://github.com/sonata-project/dev-kit/issues/380#issuecomment-360392913

Reply:
Fixed. I've updated `conflict` and `require-dev` sections